tests/tests/operation_tests.cpp

```diff
@@ -7423,7 +7424,450 @@ BOOST_FIXTURE_TEST_SUITE(auction_window_tests, votes_extended_fixture)
             generate_blocks((alice_post.cashout_time), true);
 
             share_type rwd = 0;
-            for (int i = 0; i < 20; i++) {
+            for (int i = 0; i < voters_count; i++) {
+                std::string acc_name = "voter" + std::to_string(i);
+                auto& account = db->get_account(acc_name);
+                rwd += account.curation_rewards;
+            }
+            auto& alice_acc = db->get_account(alice_post.author);
+
+            double auw_tokens = (gpo.total_reward_fund_steem.amount.value);
+            double total_payout = (alice_acc.posting_rewards.value) * 100.0 / 75.0;
+            double voters_reward = (rwd.value);
+            double voters_reward_percent = (auw_tokens + voters_reward) / total_payout;
+            double modeled_voters_reward_percent = (total_comment_fund.reward_fund().amount.value + voters_reward) / total_payout;
+
+            double allowed_percent_delta = 0.01; // = 1e-2
+            double allowed_tokens_delta = 50;
+
+            BOOST_CHECK_EQUAL(fabs(0.25 - voters_reward_percent) < allowed_percent_delta, true);
```

Comment:
APPROX_CHECK_EQUAL, no? 
And same in another places.

Reply:
`APPROX_CHECK_EQUAL` uses `std::abs` while I need `std::fabs`. Added `APPROX_CHECK_DOUBLE_EQUAL`